script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -471,33 +500,40 @@ I just had a bit of a moment."
 [sha:a60ffb96880992090c3b1012601abc4a2de33a62]{
 -- Page 314, Offset 15563.
 -- ノエル先生はこちらに背を向けて顔を隠している。
-Noel turns her back to me as to hide her face.
+Noel has turned her back to me, and is hiding her face.
 }
 [sha:308054998144ed2d48cc24bd51a7100fc0533420]{
 -- Page 314, Offset 15564.
 -- でも声だけで何を思っているかは明白だ。
-But I can clearly hear from just her voice what she's thinking.
+Even so, her voice makes it pretty clear what's on her mind.
 }
 [sha:90b4ff2d00ad3365f9e24494de2e18c511cc7b1b]{
 -- Page 314, Offset 15565.
 -- 頭隠して尻隠さず、ということわざを教えてやりたい。
-// yeah, I dunno what idiom slots into here in English. The Japanese one is quite simple "Head hidden, ass exposed".
-I really want to teach her what it means to have one's back turned.
+// P: Yeah, I dunno what idiom slots into here in English. The Japanese one is quite simple "Head hidden, ass exposed".
+// R: My dictionary lists 'burying one's head in the sand', but I don't think that's really a match here.
+// R: Interpreted much more widely as pretending something bad isn't happening than about doing a bad job of concealing yourself.
+// R: Cat's out of the bag is _close_ but doesn't have the attempt to hide, and is awk in context.
+// R: Gone with this rendering instead which is a bit different but I think _kinda_ works in that she's doing something that has no effect
+I've the urge to inform her that there's no use closing the barn door after the horse has already bolted.
```

Comment:
Hm, I don't really think this idiom works here. Even "cats out of the bag" is a bit of a stretch and would need some massaging to fit

Reply:
Yeah, it's definitely a slightly sideways TL - I think it still works in context (there's no use hiding your face / closing the barn door if the horse has bolted / I already know you're laughing), but it's not 1:1 with the JP meaning. I'll keep thinking and see if I come up with anything better.

Reply:
Yeah, I agree it's a different connotation to the JP - like I said, it's a sideways TL that makes _sense_ in EN, but doesn't carry the exact same meaning. I honestly think cat out of the bag has the same problem - the reason I didn't go with it is it doesn't encode that she's trying to keep the cat _in_ the bag, whereas the barn door one at least captures her trying to hide her face.

Reply:
Ran it by some other people, what we currently have makes sense to them but they also suggested
> I've the urge to inform her that there's no use in pissing against the wind.

Which might be a bit base, but does encode doing something that literally backfires on you

Reply:
I don't think sticking to an idiom is worth it either

Reply:
pissin' in the wind is definitely closer but I agree with Petrikow and Yusuke. I think just drop the idiom altogether, its loose and kinda weak.